lib/workers/repository/update/pr/automerge.spec.ts

```diff
@@ -31,6 +32,12 @@ describe('workers/repository/update/pr/automerge', () => {
       expect(platform.mergePr).toHaveBeenCalledTimes(0);
     });
 
+    it('should not automerge if off schedule', async () => {
+      jest.spyOn(schedule, 'isScheduledNow').mockReturnValueOnce(false);
```

Comment:
same as other test

Reply:
while indeed ive missed this in `branch/pr`
this one does have a `afterEach` block:

```
afterEach(() => {
      jest.clearAllMocks();
    });
```

Reply:
please move the `jest.spyOn(schedule, 'isScheduledNow')` to module scope, so it's more visible to others and the spy can be reused.